### DISCUSSION TITLE
Can't transcribe a longer interview :(

### DISCUSSION THREAD

**Discussion by Pompey21** [openai/whisper#500] [Q&A]
Hi guys, I am trying to use Whisper for transcribing an interview (circa 30min long) but I only get the first 10 seconds of it - not even the first 30 as stated in the README file. I would much appreciate your assistance on figuring what I am doing wrong. Here is my 2-line code:

model = whisper.load_model('base')
result = model.transcribe('/Interview-Claude-Nobs.wav')

Much appreciate your help and assistance!

**Comment by jongwook** [comment]
Does the file have a long silence in the beginning? Whisper often struggles with silent parts, and trimming those might help.

Otherwise, can you try in the command line, or using different models, like `base.en` or `small`?

**Reply by marcpope** [reply]
I trandscribe 2 hour podcasts without issue. Using a GPU

**Reply by Pompey21** [reply]
Yes there might be a short (circa 2-3s) break. So you recommend I trim these? 
Also, I cannot use GPU sadly...

**Reply by elementalcollision** [reply]
I've processed 2.5hrs of audio (almost 3 at this point) in 2 WAV files using --medium (it was a mistake) on an M1 MAX and while slow, it's chugging through.  My issue seems to be extremely poor CPU utilization.

**Reply by jongwook** [reply]
2-3s breaks are usually fine, but if the issue persists even after trimming or even with other models, please feel free to upload/email the audio file if you can. Thanks.

**Comment by FurkanGozukara** [comment]
I have a video that explains how to use whisper it may help. i install on windows from scratch and run it

https://www.youtube.com/watch?v=msj3wuYf3d8
